naff/api/events/internal.py

```diff
@@ -161,3 +162,14 @@ class Button(Component):
 @define(kw_only=False)
 class Select(Component):
     """Dispatched when a user uses a Select."""
+
+
+@define(kw_only=False)
+class Error(BaseEvent):
+    """Dispatched when the library encounters an error."""
+
+    source: str
+    error: Exception
+    args: tuple[Any] = field(factory=tuple)
+    kwargs: dict[str, Any] = field(factory=dict)
+    ctx: Optional["Context"] = field(default=None)
```

Comment:
Please can you docstring these attributes?

Reply:
I'm honestly not sure how to document args and kwargs.